server/models/UserModel.py

```diff
@@ -0,0 +1,27 @@
+from app import db
+from passlib.hash import pbkdf2_sha256 as sha256
+
+class UserModel(db.Model):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key = True)
+    email = db.Column(db.String(120), unique = True, nullable = False)
+    fName = db.Column(db.String(120))
+    lName = db.Column(db.String(120))
```

Comment:
I want to double check if the proper naming convention in SqlAlchemy for attributes is camelCase or it should be snakeCase

Reply:
I did some research, it seems that sqlalchemy naming convention for attributes is snake_case